Tasks/DownloadBuildArtifacts/main.ts

```diff
@@ -75,6 +78,44 @@ async function main(): Promise<void> {
         var artifacts = [];
         var itemPattern: string = '**';
 
+        if (isCurrentBuild)
+        {
```

Comment:
style nitpick: the opening brackets should be on the same line as the if statement to match the style of the rest of the file.